gulp-tasks/watch.js

```diff
@@ -1,21 +1,27 @@
 const gulp = require('gulp');
 
+const build = require('./build');
+
 const logHelper = require('../infra/utils/log-helper');
 
-gulp.task('watch', gulp.series(
-  'build',
+const watch = gulp.series(
+  build,
   () => {
     gulp.watch(
       [
         `packages/${global.packageOrStar}/**/*.mjs`,
         '!packages/**/_version.mjs',
         '!packages/**/build/**/*',
         '!packages/**/node_modules/**/*',
-      ], gulp.series('build'),
+      ], build,
     )
+    // GULP: why is this here?
```

Comment:
I think the idea is that during a watch, the build can fail, just watch for the next code change to fix it.

Reply:
Does the latest gulp stop the watcher if an error happens?  I didn't think it did but I might have to take another look.

Reply:
This could easily have been copy and pasted and assumed to be correct.

If the latest watcher doesn't stop that would make me soooo happy.